backend/src/media/index.ts

```diff
@@ -4,8 +4,10 @@ import { IMAGE } from 'wildebeest/backend/src/activitypub/objects/image'
 import type { Object } from 'wildebeest/backend/src/activitypub/objects'
 
 export function fromObject(obj: Object): MediaAttachment {
-	if (obj.type === IMAGE) {
+	if (IMAGE.toLowerCase() === obj.type.toLowerCase()) {
```

Comment:
In our dummy data and from mastodon the type is `image` but here we were only handling `Image`... not sure what's up with this 😕 🤷

Reply:
It should be capitalized Image here because it's an ActivityOub object. Mastodon uses lower case image as media type in their API but should use that code path. We need a convert first

Reply:
reverted, I'm converting our dummyData statuses instead